collections.go

```diff
@@ -125,23 +127,49 @@ func (cl *CollectionsListClient) Do(ctx context.Context, fn CollectionsListFn) e
 	}
 }
 
-type CollectionsCreateClient struct {
-	sl *sling.Sling
+// collectionsCreateParams represents the Outline Collections.create parameters
+type collectionsCreateParams struct {
+	Name        string     `json:"name"`
+	Description string     `json:"description,omitempty"`
+	Permission  Permission `json:"permission,omitempty"`
+	Color       string     `json:"color,omitempty"`
+	Private     bool       `json:"private,omitempty"`
 }
 
-func newCollectionsCreateClient(sl *sling.Sling, name string) *CollectionsCreateClient {
-	data := struct {
-		Name string `json:"name"`
-	}{Name: name}
+type CollectionsCreateClient struct {
+	sl     *sling.Sling
+	params collectionsCreateParams
+}
 
+func newCollectionsCreateClient(sl *sling.Sling, params collectionsCreateParams) *CollectionsCreateClient {
 	copy := sl.New()
-	copy.Post(common.CollectionsCreateEndpoint()).BodyJSON(&data)
+	return &CollectionsCreateClient{sl: copy, params: params}
+}
```

Comment:
second parameter should be name only and while returning the  new `CollectionsCreateClient` give it a new param with name field initialized from the name parameter

Reply:
Fixed here https://github.com/ioki-mobility/go-outline/pull/29/commits/bee03ffdc515c91f871b68f2a7b78d4085f2b399